game/game.go

```diff
@@ -573,3 +582,9 @@ func generateUnknownBoard() [][]string {
 	}
 	return board
 }
+
+var taunts = []string{
+	"I think you should resign.",
+	"This just isn't your game.",
+	"Do you still think you can win?",
```

Comment:
I know I'm late to the party, but...

https://memegenerator.net/img/instances/x300/74907332/good-heavens-are-you-still-trying-to-win.jpg